JASP-Desktop/components/JASP/Theme/Theme.qml

```diff
@@ -33,22 +33,22 @@ JaspTheme
 
 	fontLabel.bold:										true
 	fontLabel.pixelSize:								Math.round(18 * uiScale)
-	fontLabel.family:									jaspFont + (MACOS ? " Bold" : "")
+	fontLabel.family:									jaspFont
```

Comment:
Did you make sure the font s are still bold/demibold etc when they ought to be?

Reply:
I see no difference on my Mac if I remove this. It seems anyway quite redundant with the bold property.
And it must be removed because most of the fonts do not have a 'Bold' version: then it falls back to the default font.

Reply:
I think you would see the difference if you would specify the fonts as QFont instead of as QString, but if it works it works so whatever.